string_scrap.js

```diff
@@ -1,31 +1,43 @@
+//1
 let word = 'PURSUIT';
 console.log(word[0]);
 
+//2
 let str = 'PURSUIT';
 console.log(str.length);
 
+//3
 let name = 'PURSUIT';
 let lastCHar = name.charAt(name.length - 1);
 console.log(lastCHar);
 
+//4
 let name1 = 'jimmy';
 let lastCHar2 = name1.charAt(name1.length - 1);
 console.log(lastCHar2);
 
+//5
 let drEvil = '10';
 if (drEvil[5] === '0' || drEvil [2] === '0') {
```

Comment:
Although very interesting approach to solve this problem, if we change drEvil to 40, it will give us output of "10 dollars". This is instance of hard coding which is a bad practice. Your code must be able to accommodate for differing inputs and still give us consistent outputs.